cEP-0009.md

```diff
@@ -0,0 +1,181 @@
+| Metadata     |                                                          |
+| ------------ |----------------------------------------------------------|
+| cEP          | 9                                                        |
+| Version      | 0.1                                                      |
+| Title        | Extract and utilize information from project files       |
```

Comment:
@adtac It's aligned properly.
![1495724752](https://cloud.githubusercontent.com/assets/18437312/26456121/5dc5630e-4196-11e7-9da0-c6ed3d86c89f.png)